js/pro/huobi.js

```diff
@@ -512,17 +475,56 @@ module.exports = class huobi extends huobiRest {
         //         },
         //         "ts":1645023376098
         //     }
+        // non-spot market snapshot
         //
+        //     {
+        //         "ch":"market.BTC220218.depth.size_150.high_freq",
+        //         "tick":{
+        //             "asks":[
+        //                 [43445.74,1],
+        //                 [43444.48,0 ],
+        //                 [40593.92,9]
+        //             ],
+        //             "bids":[
+        //                 [43445.74,1],
+        //                 [43444.48,0 ],
+        //                 [40593.92,9]
+        //             ],
+        //             "ch":"market.BTC220218.depth.size_150.high_freq",
+        //             "event":"snapshot",
+        //             "id":152727500274,
+        //             "mrid":152727500274,
+        //             "ts":1645023376098,
+        //             "version":37536690
+        //         },
+        //         "ts":1645023376098
+        //     }
+        //
+        const ch = this.safeValue (message, 'ch');
+        const parts = ch.split ('.');
+        const marketId = this.safeString (parts, 1);
+        const symbol = this.safeSymbol (marketId);
         const tick = this.safeValue (message, 'tick', {});
-        const seqNum = this.safeInteger2 (tick, 'seqNum', 'id');
+        const seqNum = this.safeInteger2 (tick, 'seqNum', 'version');
         const prevSeqNum = this.safeInteger (tick, 'prevSeqNum');
+        const event = this.safeString (tick, 'event');
+        const timestamp = this.safeInteger (message, 'ts');
+        if (event === 'snapshot') {
+            const snapshot = this.parseOrderBook (tick, symbol, timestamp);
+            orderbook.reset (snapshot);
+            orderbook['nonce'] = seqNum;
+        }
+        if (event === 'update') {
```

Comment:
@pcriadoperez since now we get everything from the WS channel, I don't think we need these verifications